addons/crm/static/src/new/activity_menu_patch.js

```diff
@@ -0,0 +1,13 @@
+/* @odoo-module */
+
+import { ActivityMenu } from "@mail/new/web/activity/activity_menu";
+import { patch } from "@web/core/utils/patch";
+
+patch(ActivityMenu.prototype, "crm", {
+    get availableView() {
+        if (this.model === "crm.lead") {
+            return ["list", "kanban", "form", "calendar", "pivot", "graph", "activity"];
```

Comment:
format should be the same as parent `[false, item]` (or the caller should do this map like in old code)